README.md

```diff
@@ -6,6 +6,12 @@ Website of the Czech PyLadies chapter / Web českých PyLadies.
 
 *Pokud je kontrolka zelená, povedlo se dostat poslední změny z větve `master` do ostrého provozu. Má-li jinou barvu, stala se [někde po cestě](https://travis-ci.org/PyLadiesCZ/pyladies.cz) chyba.*
 
+
+## License
```

Comment:
to jo, ta vlajka to je komentar. Ale ta cast v sedive tabulce je navrh na zmenu, ktery po merge patri ke zmenam a nebo je to jen komentar napsany v MD?